### PR TITLE
[Merged by Bors] - fix: reduce imports for scripts

### PR DESCRIPTION
**Pull request by digama0**
As [noted on Zulip](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/lake.20exe.20cache.20get.20broken.3F/near/386489865), a from-scratch build of mathlib after `lake exe cache get` will compile all of `Std` due to some unnecessary imports. With a few well chosen import reductions we only end up having to compile ~20 files instead of ~300 files (compile meaning `Compiling`, generating the arch-dependent `.o` files that are not in the cache).

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by eric-wieser — APPROVED** [review]
bors d+

Thanks!

File: scripts/checkYaml.lean [hunk 1]
```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Eric Wieser
 -/
 
-import Mathlib.Tactic.Basic
+import Std.Tactic.Lint
```
Comment (eric-wieser) [inline scripts/checkYaml.lean:7]: This one seems strange to me; is it really minimal, despite this file not being about linting?

Comment (digama0) [reply scripts/checkYaml.lean:7]: It has a `open Std.Tactic.Lint` in it, I didn't investigate why. This import is not precisely minimal, `Std.Tactic.Lint` is the root of a small number of other files

Comment (digama0) [reply scripts/checkYaml.lean:7]: Apparently the `open Std.Tactic.Lint` was unnecessary and after removing it the `import Std.Tactic.Lint` could be removed as well.

**Pull request closed without merging**